light-client/src/main/java/tech/pegasys/teku/lightclient/client/Types.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.lightclient.client;
+
+import tech.pegasys.teku.spec.logic.versions.altair.SpecLogicAltair;
+
+public class Types {
+	
+	
+	
+	private SpecLogicAltair header;
```

Comment:
It's very unusual to need to access any of the spec logic classes directly.  Normally interactions should go through the `Spec` class so it can ensure you have the right logic for the active fork. Sync committees will apply to the merge fork as well as Altair for example and this will need to correctly handle chains that are still in phase0.

Reply:
Which is why I closed this. I was in the process of building/deleting this (just figured out it got implemented into this pull request when I pushed the local repo up to the remote repo).

Reply:
Yep all good, just wanted to give as much feedback as possible as early as possible since that's better than heading off down a big rabbit hole.